src/app/static/src/app/apiService.ts

```diff
@@ -15,7 +15,7 @@ export interface ResponseWithType<T> extends Response {
 export interface API<S, E> {
 
     withError: (type: E) => API<S, E>
-    withSuccess: (type: S) => API<S, E>
+    withSuccess: (type: S, root: boolean) => API<S, E>
```

Comment:
Added this option so I could use a root mutation as the success mutation for a new version, to reset the entire root state (without it, Vuex assumes the named mutation is relative to the home module, versions).